README.md

```diff
@@ -167,12 +188,28 @@ and user merge event in one request, to make them taken into account when execut
 ```php
 $matej = new Matej('accountId', 'apikey');
 
-$response =  $matej->request()
+$sortedData =  $matej->request()
     ->sorting(Sorting::create('user-id', ['item-id-1', 'item-id-2', 'item-id-3']))
     ->setInteraction(Interaction::purchase('user-id', 'item-id')) // optional
     ->setUserMerge(UserMerge::mergeInto('user-id', 'source-id')) // optional
+    ->send()
+    ->getSorting()
+    ->getData();
```

Comment:
And this could be 

```
$response =  $matej->request()
     ->sorting(Sorting::create('user-id', ['item-id-1', 'item-id-2', 'item-id-3']))
     ->setInteraction(Interaction::purchase('user-id', 'item-id')) // optional
     ->setUserMerge(UserMerge::mergeInto('user-id', 'source-id')) // optional
     ->send();

$sortedItems  = $response->getSorting()
    ->getData();
```

Your thoughts :)?